### ISSUE TITLE
stringify

### ISSUE THREAD

**Issue by jsannerstedt** [BigstickCarpet/simplifyify#17]
When using the following

"browserify": {
    "transform": [
      "babelify",
      [
        "stringify",
        {
          "extensions": [
            ".html"
          ]
        }
      ]
    ]
  }

my bundle has \\n instead of \n, in all my .html files, and any escaped chars have \\\. So there seems to be one too many back slashes throughout my templates. 

Any idea or suggestion how I could fix this?

(works as expected with watchify)

**Comment by BigstickCarpet** [comment]
This sounds like a bug in the stringify transform.  Maybe something related to the interaction between stringify and babelify.  Try it with Browserify directly (rather than going through Simplifyify) and I suspect that you'll see the same behavior

**Comment by jsannerstedt** [comment]
nope, works fine with both browserify and watchify, exact same args and files
